web/web.go

```diff
@@ -816,7 +823,7 @@ func (h *Handler) runtimeInfo() (api_v1.RuntimeInfo, error) {
 
 	metrics, err := h.gatherer.Gather()
 	if err != nil {
-		return status, fmt.Errorf("error gathering runtime status: %w", err)
```

Comment:
Could you change this one back to `%w`? It's showing this linter annotation:

![image](https://github.com/user-attachments/assets/661466d2-e917-4677-8260-c30f70c4f005)